github/ci/prow-deploy/files/jobs/kubevirt/kubevirt-tekton-tasks/tekton-tasks-periodics.yaml

```diff
@@ -0,0 +1,39 @@
+periodics:
+  - name: periodic-update-libguestfs-container-version
+    cron: 0 2 * * *
```

Comment:
does this need to run so regularly? 

maybe weekly is enough?

Reply:
I already changed it to weekly, but I forget to push it to GH :)